vr-wg-charter.html

```diff
@@ -287,18 +282,18 @@ <h3>
             specifications:
           </p>
           <dl>
-            <dt id="webvr" class="spec">
-              <a href="#">WebVR API</a>
+            <dt id="webxr" class="spec">
+              <a href="#">WebXR Device API</a>
             </dt>
             <dd>
               <p>
                 This specification describes support for accessing virtual
-                reality (VR) devices, including sensors and head-mounted
-                displays on the Web.
+                reality (VR) and augmented reality (AR) devices,
+                including sensors and head-mounted displays on the Web.
               </p>
               <p class="draft-status">
-                <a href="https://w3c.github.io/webvr/spec/latest/">"v2.0"
-                adopted from WebVR CG</a>
+                <a href="https://w3c.github.io/webvr/spec/latest/">"1.0"
```

Comment:
This one is 404, I think we want:

`<a href="https://immersive-web.github.io/webxr/">Adopted from Immersive Web CG</a>`

Applying versioning in this context might be slightly confusing. I believe we want whatever is the latest version at the time of migration to get adopted.